src/main/java/no/nav/foreldrepenger/selvbetjening/vedlegg/ImageScaler.java

```diff
@@ -85,7 +89,7 @@ public Matrix transform(PDPage pdPage) {
         private Bildedimensjon roterOgSkalerNed(float pageSizeWidth, float pageSizeHeight) {
             var skalRoteres = width > height && width > pageSizeWidth;
             if (skalRoteres) {
-                return new Bildedimensjon(height, width, true).skalertDimensjon(pageSizeWidth, pageSizeHeight);
+                return new Bildedimensjon(width, height, true).skalertDimensjon(pageSizeWidth, pageSizeHeight);
```

Comment:
Bildedimensjoner har følgende konstruktør:
        private Bildedimensjon(float width, float height, boolean rotert) {
            this.width = width;
            this.height = height;
            this.rotert = rotert;
        }

Reply:
@dijjal Kunne du bekreftelse på at du er enig i at dette er riktig? :)

Reply:
Stryk denne